tools/intensity_check/xml_intensity_check.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="intensity_check" name="Intensity Check" version="@TOOL_VERSION@" profile="22.05">
+<tool id="intens_check" name="Intensity Check" version="@TOOL_VERSION@" profile="22.05">
```

Comment:
This will break tool linage. That means you will see two version of the tool in your history and they are not connected. Automatic workflow updates will also not work.

Reply:
This is actually the initial problem. The original id is "intens_check", and somehow it turned to "intensity_check" during the migration, leading to the "two version" thing. So the idea is to turn back to the correct id.